### ISSUE TITLE
how to set hyper-parameter

### ISSUE THREAD

**Issue by lipanpeng** [JiahuiYu/generative_inpainting#21]
Although you have released pretrained models, I want to train a model by myself with the celeba_hq dataset. With images of size 1024*1024, I set the hyper-parameter
IMG_SHAPES: [256, 256, 3]
HEIGHT: 128
WIDTH: 128
MAX_DELTA_HEIGHT: 64
MAX_DELTA_WIDTH: 64
RANDOM_CROP: False
other hyper-parameters same as yours in inpaint.yml,
but I can't get a good model like yours.
Can you give me some advice with the hyper-parameter?

**Comment by lipanpeng** [comment]
Thank you for reply. I just trained the model for half a day. It explains why I get a bad model. With the celeba_hq dataset, you resize the image of size 1024*1024 to 256*256. I wonder if it cause some information missing.

**Comment by Cristy94** [comment]
I tried on my own dataset, on a bad GPU (1050ti laptop), and after about 8 hours of training the model seemed to converge to a pretty bad model (loss was the same for several epochs). The overall model generate a good infill approximation, but the result was blurry. I used 64x64 images to speed-up training. Do you think that the model would improve if I keep training it more, or if the loss doesn't change for several epochs  than it will never get better?

**Comment by JiahuiYu** [comment]
@lipanpeng Hi, training 1024x1024 will require significant more efforts, as shown in paper: [Progressive Growing of GANs for Improved Quality, Stability, and Variation](https://github.com/tkarras/progressive_growing_of_gans). This project aims a resolution around 256x256.

@Cristy94 If the result is blurry, I hypotheses that the pixel-wise loss is too large. Please consider lower pixel-wise loss weights, e.g., set 0.5 to `COARSE_L1_ALPHA`, `L1_LOSS_ALPHA`, `AE_LOSS_ALPHA`.

**Comment by Cristy94** [comment]
Can the the loss parameters be changed after the model was created (eg. training was started) ?

**Comment by JiahuiYu** [comment]
Yes you can. Simply change the hyper-parameters, and relaunch with [restore](https://github.com/JiahuiYu/generative_inpainting_private/blob/00972d477ec7d592b56183e16ec4ceec4b31901e/inpaint.yml#L6). New training will save to new model/log files.

**Comment by AvisekNaug** [comment]
> Can the the loss parameters be changed after the model was created (eg. training was started) ?

@Cristy94 were you able to remove the blurr after reducing the pixel wise loss weights?

**Comment by Cristy94** [comment]
> > Can the the loss parameters be changed after the model was created (eg. training was started) ?
> 
> @Cristy94 were you able to remove the blurr after reducing the pixel wise loss weights?

Late response, but no, the blur was still there, so I stopped training. I am now using the new v2 model and hopefully I will get better results.
